src/Twig/Extension/TemplateExtension.php

```diff
@@ -11,7 +11,7 @@
 
 namespace Sonata\Twig\Extension;
 
-use Sonata\CoreBundle\Model\Adapter\AdapterInterface;
+use Sonata\Doctrine\Adapter\AdapterChain;
```

Comment:
`AdapterChain` implement now the AdapterInterface located in `Sonata\Doctrine\Adapter\AdapterInterface`. 

I think you should use it instead directly it's implementation. WDYT ?

```suggestion
use Sonata\Doctrine\Adapter\AdapterInterface;
```